monero-outreach-docs/README_i18n.md

```diff
@@ -6,33 +6,45 @@ Welcome to the Monero Outreach Localization. This directory contains all the fin
+4. **File Formats** - Documents are in .sla file format, so you will need a application that supports .sla files or download [Scribus](https://www.scribus.net/) (its an open source app available for different OS) or use any text-editor to work on the .sla file.
+ - When adding a new language, you might encounter that the text doesn't fit, feel free to edit the file. Try to maintain the format so the changes won't affect its structure. Design edits will be done once you submit your work. If you need help, please get in contact with us.
+5. When saving changes, remember to add a message to the commit with a description of **the documents name and the language** (e.g. **_"Quickfacts translted to fr"_**, **_"Missing punctuation donttalk PSA pt_Br"_**, etc.).
+6. If you are a reviewer please visit, [Taiga EPIC #69 Monero Outreach Localization](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/epic/69) to get assigned a translation.
+7. Happy contributing! 
 
 ### Fonts
 
 Fonts can be found inside monero-outreach-docs folder or directly in the site of Monero Outreach - [Monero Typography](https://www.monerooutreach.org/monero-typography.php) Keep in mind that some of the fonts are missing from the website, take a look at the `fonts` folder.  
 
 ## Need Help?
 
- - For support and coordination - `#monero-translations`/ IRC/ [Mattermost](https://mattermost.getmonero.org/monero/channels/monero-translations)
- - This repository is connected with the [Monero Outreach - Taiga instance](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/). If you encounter any trouble leave your comments [here](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/us/48?kanban-status=317).
- - This repository is maintained by  the Communication Workgroup Coordinator - lh1008, the coordinator of the Monero Localization Workgroup, erciccione and the organizer of the Outreach workgroup - xmrhaelan. 
-	* Email: paintluis[@]protonmail.com, monerooutreach[@]protonmail.com, translate@getmonero.org, erciccione[@]protonmail.com.
- - Monero Outreach workgroup [Telegram Join Link](https://t.me/joinchat/DM8-zRAVi-tEx-1PEltAIg).
-	* Telegram user [@]lh1008.
+ - For support and coordination - `#monero-translations`/ IRC/ Matrix/ Riot/ [Mattermost](https://mattermost.getmonero.org/monero/channels/monero-translations) or telegram [@outreachtranslations](https://t.me/outreachtranslations).
+ - This repository is connected with the [Monero Outreach - Taiga instance](https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/).
+ - This repository is maintained by:
+   - [lh1008](https://github.com/lh1008) - Monero Outreach Communication Workgroup Coordinator
+   - [erciccione](https://github.com/erciccione) - Monero Localization
+   - [BlackLotus64](https://github.com/BlackLotus64) - Project Support
```

Comment:
Maintainers are defined roles held by who have push access to the repository. BlackLotus64 has no push access and should be removed from this list

Reply:
Access to a repository is not something that should be given so easily. Maintainers have the responsability to keep the repo secure and well organized. This repo has already 3 maintainers, and that's more than enough. Monero, monero-gui and monero-site, for example have many maintainers, but all the merging activity is done by just one person for each project. To not overlap workflows.

Also, a maintainer should be able to give support to contributors about every part of the workflow. This means that should be able to give support in case of conflicts and make sure that the git history is clean and readable. Beside the fact that another maintainer is not needed and would complicate the workflow, i don't think blacklotus has the experience necessary for this role.

Reply:
ok thanks :).